Assets/HoloToolkit/Input/Scripts/InputSources/TouchscreenInputSource.cs

```diff
@@ -0,0 +1,238 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using System.Collections.Generic;
+
+namespace HoloToolkit.Unity.InputModule
+{
+    /// <summary>
+    /// Input source supporting basic touchscreen input:
+    /// * taps
+    /// * holds
+    /// Note that a hold-started is raised as soon as a contact exceeds the Epsilon value;
+    /// if the contact subsequently qualifies as a tap then a hold-cancelled is also raised.
+    /// </summary>
+    public class TouchscreenInputSource : BaseInputSource
+    {
+        const float kContactEpsilon = 2.0f/60.0f;
+
+        [SerializeField]
+        [Tooltip("Time in seconds to determine if the contact registers as a tap or a hold")]
+        protected float MaxTapContactTime = 0.5f;
+
+        private List<PersistentTouch> ActiveTouches;
```

Comment:
you could initialize this field in line and remove the start method all together.

`private List<PersistentTouch> ActiveTouches = new List<PersistentTouch>(0);`